test/Serialization/Fields.jl

```diff
@@ -0,0 +1,16 @@
+@testset "Fields" begin
+    mktempdir() do path
+        @testset "Field Embeddings" begin
+            Qx, x = QQ["x"]
+            K, _ = NumberField(x^2 + 5)
+            E = complex_embeddings(K)[1]
+
+            test_save_load_roundtrip(path, E) do loaded
+                loaded_K = number_field(loaded)
+                g = gen(loaded_K)
+                @test contains(loaded(g), E(gen(K)))
+                @test contains(E(gen(K)), loaded(g))
```

Comment:
```suggestion
                @test overlaps(loaded(g), E(gen(K)))
                @test overlaps(E(gen(K)), loaded(g))
```
This should be a bit more robust.